js/test/fixtures/pytorch.js

```diff
@@ -1,85 +1,175 @@
 import * as tf from '@tensorflow/tfjs';
 import Command from '../../src/command';
 
-export const add = new Command(
-  'torch.add(input,other,out=None',
-  'add',
-  [tf.tensor([1]), tf.tensor([1])],
-  [['out', 'None']]
-);
+export const add = {
+  inputs: [
```

Comment:
East text fixture has `inputs` and `answers` (of the same length) which will be compared to each other in our test suite.